com.unity.render-pipelines.universal/Shaders/Lit.shader

```diff
@@ -332,8 +332,9 @@ Shader "Universal Render Pipeline/Lit"
             #pragma target 4.5
 
             #pragma vertex UniversalVertexMeta
-            #pragma fragment UniversalFragmentMeta
+            #pragma fragment UniversalFragmentMetaLit
 
+            #pragma shader_feature EDITOR_VISUALIZATION
```

Comment:
This is only needed in editor, how is this keyword stripped?

Reply:
How can I make sure it gets stripped from builds?

Reply:
I'm not sure what's the behaviour regarding shader_feature that's not references in any material, I guess it's stripped automatically. @erikabar maybe you can confirm this.

Reply:
It looks like you have to explicitly strip it. Kuba added it to his PR for stripping lighting-related keywords here: https://ono.unity3d.com/unity/unity/pull-request/127683/_/lighting/bugfix/1307896-dirlightmap-stripping